scripts/update-doc-yml.py

```diff
@@ -0,0 +1,38 @@
+"""
+Author: Jack He (jackhe@microsoft.com)
```

Comment:
I'd hide my corp email unless you wanna get spammed. There's no need to add a line for authorship.

Reply:
Oof